src/System.CommandLine/Builder/CommandLineBuilder.cs

```diff
@@ -68,19 +82,19 @@ HelpBuilder CreateHelpBuilder(BindingContext bindingContext)
             }
         }
 
-        internal HelpOption? HelpOption { get; set; }
+        internal HelpOption? HelpOption;
```

Comment:
fields: no need to JIT a property. A micro optimization for internal properites only